openbas-api/src/main/java/io/openbas/executors/Injector.java

```diff
@@ -54,7 +55,7 @@ public void setFileService(FileService fileService) {
   public abstract ExecutionProcess process(Execution execution, ExecutableInject injection)
       throws Exception;
 
-  public InjectStatusCommandLine getCommandsLines(String externalId) {
+  public PayloadOutput getPayloadOutput(String externalId) {
     return null;
   }
```

Comment:
@damgouj Why are we returning null here (we pass a param but it's never used)? Couldn't we at least use Optional?

Reply:
This method is used to translate the third parties systems  approach of payloads  into openBAS payloads modeling. For now it is only implemented for Caldera.